packages/react-mutation-mapper/src/util/LabelUtils.tsx

```diff
@@ -0,0 +1,36 @@
+export function labelDisplayText(
+    label: string,
+    textElt: SVGTextElement | null,
+    width: number
+) {
+    // Truncate text if necessary
+    if (!textElt) {
+        return label;
+    }
+
+    if (!$(textElt!).is(':visible')) {
```

Comment:
I think this should also work:

```suggestion
    if (!$(textElt).is(':visible')) {
```